json_schemas/users.json

```diff
@@ -22,6 +22,11 @@
       "type": "string",
       "pattern": "^(buyer|supplier|admin)$"
     },
+    "supplier_id": {
+      "type": "string",
```

Comment:
was a bit worried given our travails with the service_id. so we want to commit to integer IDs?